nagios/README.md

```diff
@@ -22,17 +22,37 @@ host_perfdata_file_processing_command=wavefront_perf_host
 ```
 define command {
     command_name  wavefront_perf_host
-    command_line  /opt/nagios/etc/wf/nagios-metrics-wf.py [PATH_PERFDATA]/host-perfdata \
+    command_line  /usr/local/nagios/libexec/nagios-metrics-wf.py [PATH_PERFDATA]/host-perfdata \
                                                            --wf_proxy_addr [WAVEFRONT_PROXY_ADR] \
                                                            --wf_proxy_port [WAVEFRONT_PROXY_PORT]
+    command_line /usr/local/nagios/libexec/nagios-metrics-wf.py [PATH_PERFDATA]/host-perfdata \
+                                                           --wf_server [WAVEFRONT_URL] \
+                                                           --csp_api_token [CSP_API_TOKEN] \
+                                                           --csp_base_url [CSP_BASE_URL]
+    command_line /usr/local/nagios/libexec/nagios-metrics-wf.py [PATH_PERFDATA]/host-perfdata \
+                                                           --wf_server [WAVEFRONT_URL] \
+                                                           --csp_app_id [CSP_APP_ID] \
+                                                           --csp_app_secret [CSP_APP_SECRET] \
+                                                           --csp_base_url [CSP_BASE_URL]                                                           
 }
 
 define command {
     command_name  wavefront_perf_service
-    command_line  /opt/nagios/etc/wf/nagios-metrics-wf.py [PATH_PERFDATA]/service-perfdata \
+    command_line  /usr/local/nagios/libexec/nagios-metrics-wf.py [PATH_PERFDATA]/service-perfdata \
```

Comment:
this will change if we are having multiple files with single command OR single command and rest as example?

Reply:
Yes. I'll change the Readme also